packages/@expressive-code/plugin-collapsible-sections/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import { AttachedPluginData, ExpressiveCodePlugin, replaceDelimitedValues } from '@expressive-code/core'
+import { Section, parseSections } from './utils'
+import { select } from 'hast-util-select'
+import { SectionText, sectionizeAst } from './ast'
+import { collapsibleSectionsStyleSettings, getCollapsibleSectionsBaseStyles } from './styles'
+
+export interface PluginCollapsibleSectionsOptions {
+	styleOverrides?: Partial<typeof collapsibleSectionsStyleSettings.defaultSettings> | undefined
+	summary?: SectionText | undefined
+}
+
```

Comment:
```suggestion
import { AttachedPluginData, ExpressiveCodePlugin, PluginTexts, replaceDelimitedValues } from '@expressive-code/core'
import { Section, parseSections } from './utils'
import { select } from 'hast-util-select'
import { sectionizeAst } from './ast'
import { collapsibleSectionsStyleSettings, getCollapsibleSectionsBaseStyles } from './styles'

export interface PluginCollapsibleSectionsOptions {
	styleOverrides?: Partial<typeof collapsibleSectionsStyleSettings.defaultSettings> | undefined
}

export const pluginCollapsibleSectionsTexts = new PluginTexts({
	collapsedLines: (count: number) => `${count} collapsed line${count === 1 ? '' : 's'}`,
})

pluginCollapsibleSectionsTexts.addLocale('de', {
	collapsedLines: (count: number) => `${count} ausgeblendete Zeile${count === 1 ? '' : 'n'}`,
})

```

Please use the existing `PluginTexts` API for i18n purposes. It allows type-safe language-specific interpolation for cases like this which need to incorporate a line counter into the text. It also allows plugins to define localized texts without taking away the user's ability to add their own or override existing ones.